src/data-loader.ts

```
@@ -12,8 +14,18 @@ import {
 } from './pricing-fetcher.ts';
 import { groupBy } from './utils.internal.ts';
 
+const DEFAULT_CLAUDE_CODE_PATH = path.join(homedir(), '.claude');
+
+/**
+ * Default path for Claude data directory
+ * Uses environment variable CLAUDE_CONFIG_DIR if set, otherwise defaults to ~/.claude
+ */
 export function getDefaultClaudePath(): string {
-	return path.join(homedir(), '.claude');
+	const envClaudeCodePath = process.env.CLAUDE_CONFIG_DIR?.trim() ?? '';
```

ESLint

[error] Unsafe assignment of an error typed value.

(ts/no-unsafe-assignment)